client/server/pages/index.js

```diff
@@ -123,6 +124,13 @@ function getDefaultContext( request, response, entrypoint = 'entry-main' ) {
 	const cachedServerState = request.context.isLoggedIn ? {} : stateCache.get( cacheKey ) || {};
 	const getCachedState = ( reducer, storageKey ) => {
 		const storedState = cachedServerState[ storageKey ];
+
+		logServerEvent( request.context.sectionName, {
```

Comment:
The `request.context.sectionName` is `undefined` here, probably because this is [called](https://github.com/Automattic/wp-calypso/blob/e538d107e7fc0741e64604797f2fec642bf2c5e3/client/server/pages/index.js#L820) before [setUpSectionContext](https://github.com/Automattic/wp-calypso/blob/e538d107e7fc0741e64604797f2fec642bf2c5e3/client/server/pages/index.js#L524).
This triggers an error [here](https://github.com/Automattic/wp-calypso/blob/e538d107e7fc0741e64604797f2fec642bf2c5e3/client/lib/analytics/statsd-utils.ts#L36) and I expect the e2e are failing because of this.

Reply:
Thanks for looking into this!